src/components/layout/layout.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { ReactNode, Ref } from 'react'
+import React, { useState, ReactNode, Ref } from 'react'
```

Comment:
👀

Reply:
I forgot to create the PR as draft this is still work in progress but thanks for the heads up @NikitK-deriv